app/policies/rubygem_policy.rb

```diff
@@ -24,6 +24,26 @@ def resolve
   has_association :link_verifications
   has_association :oidc_rubygem_trusted_publishers
 
+  def show?
+    true
+  end
+
+  def create?
+    user.present?
+  end
+
+  def update?
+    false
+  end
+
+  def destroy?
+    false
+  end
+
+  def show_ownerships?
+    record.owned_by?(user)
+  end
```

Comment:
When I see this now, it doesn't really make sense. Anyone should be able to see ownerships, but maybe not "unconfirmed" ownerships. I should change it to `show_unconfirmed_ownerships?`.